USERGUIDE.md

```diff
@@ -239,6 +241,36 @@ extension ViewController: MiniAppMessageDelegate {
 }
```

Comment:
Please mention in the doc that getUniqueId() will be deprecated

Reply:
```NOTE:  This method `getUniqueId(completionHandler:)` will be deprecated in the upcoming release of MiniAppSDK. Please use `getMessagingUniqueId(completionHandler:)` instead.```